internal/pkg/service/stream/plugin/collection.go

```diff
@@ -70,7 +151,7 @@ func (c *Collection) OnSourceDelete(fn onSourceSaveFn) {
 
 func (c *Collection) OnSourceUndelete(fn onSourceSaveFn) {
 	c.onSourceSave = append(c.onSourceSave, func(ctx context.Context, now time.Time, by definition.By, original, branch *definition.Source) error {
```

Comment:
`branch` used as `definition.Source`. Expecting that `source` is `onUndeleted` not `branch`.

Reply:
https://github.com/keboola/keboola-as-code/pull/1792/commits/746f67df64188dc4de776baa4858614f378da6c1